public/app/plugins/panel/xychart/module.tsx

```diff
@@ -0,0 +1,225 @@
+import {
+  FieldColorModeId,
+  FieldConfigProperty,
+  FieldType,
+  identityOverrideProcessor,
+  PanelPlugin,
+} from '@grafana/data';
+import {
+  AxisPlacement,
+  DrawStyle,
+  GraphFieldConfig,
+  graphFieldOptions,
+  LegendDisplayMode,
+  PointVisibility,
+  ScaleDistribution,
+  ScaleDistributionConfig,
+} from '@grafana/ui';
+import { SeriesConfigEditor } from '../graph3/HideSeriesConfigEditor';
+import { XYChartPanel } from './XYChartPanel';
+import { defaultXYDimensions, Options } from './types';
+import { ScaleDistributionEditor } from '../graph3/ScaleDistributionEditor';
+import { XYDimsEditor } from './XYDimsEditor';
+
+export const plugin = new PanelPlugin<Options, GraphFieldConfig>(XYChartPanel)
+  .useFieldConfig({
+    standardOptions: {
+      [FieldConfigProperty.Color]: {
+        settings: {
+          byValueSupport: false,
+        },
+        defaultValue: {
+          mode: FieldColorModeId.PaletteClassic,
+        },
+      },
+    },
+    useCustomConfig: builder => {
```

Comment:
would be nice to extract this and share the common definitions

Reply:
done ;)